### ISSUE TITLE
Requested tag: Web Services

### ISSUE THREAD

**Issue by iherman** [w3c/tr-pages#58]
The current list of documents includes a large number of specs and notes around SOAP and Web Services in general. Although they can all be labeled as XML, too, I believe a proper approach would be to add a separate tag 'Web Services", too.

Cc @plehegar @koalie

**Comment by plehegar** [comment]
What would be our description for that tag? We would probably want to describe it in opposition to our current Web API tag: "Specifications providing APIs to allow Web applications to access specific features or data".

**Comment by iherman** [comment]
I am probably not the proper person to answer this; somebody who was more deeply involved with Web Services should give an idea. It is certainly not me...

**Comment by plehegar** [comment]
I wonder if there is enough of use case here, ie how often folks come to our site to look for WS-* specs nowadays.

**Comment by iherman** [comment]
I don't think that thinking in terms of "use cases" is the right thing to do. We have treat our own history properly; this like cataloging the content of a library properly.

**Comment by plehegar** [comment]
We have a long history and I don't  think it's useful to represent our entire history in the list of tags. As Denis mentioned in the past, one goal is to keep the list of tags to a minimum (at least, as listed in the dropdown UI).
